### PR TITLE
Js disable error page

### PR DESCRIPTION
**Pull request by SannyNguyenHung**
# Short Description
- JS Disabled page created.
- Once JS is disabled, the main_content is hidden and an error message is displayed.

# Changes
- Updated localization
- Add error page
- Implemented the js error page in "base.html"

### CODE REVIEW

**Review by frederike-ramin — COMMENTED** [review]
I didn't have an exact look at the code yet. Just some things that occurred to me when I tried out the feature: 
- [x] The link to enable javascript should probably open in a new tab?
- [x] When I click on the link, I get an error that the page doesn't exist

File: webapp/app/templates/basis/base.html [hunk 1]
```diff
@@ -29,35 +29,45 @@
...
+        {% include 'basis/mobile_nav.html' %}
+        {% include 'basis/sidebar.html' %}
+        <main id="main" class="main">
+            <noscript>
+                {% include 'error/js_disabled.html' %}
+
+                <!-- If javascript is disabled, the main_content container is hidden. -->
+                <style type="text/css">
+                        #main_content { display: none!important; }
+                </style>
```
Comment (frederike-ramin) [inline webapp/app/templates/basis/base.html:54]: I don't think we ever use <style> tags so far. I just think it's interesting. Did you get the idea from somewhere? (i.e., is this the best practice way of handling disabled JavaScript that I should memorize? :D )

File: webapp/Pipfile.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
```
[the remark this answers is not in the document]
Comment (SannyNguyenHung) [reply webapp/Pipfile.lock:1]: :+1: Thank you! I will remember  :smiley:

File: webapp/app/templates/basis/base.html (hunk 1 above)
Comment (SannyNguyenHung) [reply webapp/app/templates/basis/base.html:54]: Yes, I think with a slow computer the web page might flicker. Others also have <meta> in the <noScript> container to redirect to another page. But this is probably not supported by each browser.

**Pull request merged**